File: problem-solving/3_compare-the-triplets/javascript/main.js
function compareTriplets(a, b) {
  let alice = 0;
  let bob = 0;
  if (a.length == b.length) {
    //constraints
    for (let i = 0; i < a.length; i++) {
      if (a[i] < 1 || b[i] < 1 || a[i] > 100 || b[i] > 100) {
        return;
      }
    }

    //compare
    for (let i = 0; i < a.length; i++) {
      if (a[i] > b[i]) alice++;
      else if (a[i] < b[i]) bob++;
    }
  }
  return [alice, bob];
}

//test
console.log(compareTriplets([1, 2, 3], [1, 2, 3]));

File: problem-solving/3_compare-the-triplets/python/main.py
def compareTriplets(a, b):
    # constraints
    a = [i for i in a if 1 <= i <= 100]
    b = [i for i in b if 1 <= i <= 100]

    alice = 0
    bob = 0

    if len(a) == len(b):
        for i in range(0, len(a)):
            if a[i] > b[i]:
                alice += 1
            elif a[i] < b[i]:
                bob += 1
    return [alice, bob]


# test
print(compareTriplets([5, 6, 7], [3, 6, 10]))
